src/components/DisplayColours.jsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react'
+import TextSection from './TextSection'
 
 function DisplayColours({ colors }) {
   return (
     <>
+    <TextSection />
```

Comment:
This component is not getting any properties passed in, causing the app to crash when it renders.